openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -4017,7 +4032,9 @@ def from_molecule(cls, molecule):
     _INFOTYPE = LibraryChargeType  # info type to store
     _DEPENDENCIES = [vdWHandler, ElectrostaticsHandler]
 
-    def find_matches(self, entity):
+    _KWARGS = ["unique"]
```

Comment:
(not blocking) I don't think this should be here -- It's a list of strings that tells `ff.create_openmm_system(args, kwargs)` which kwargs are consumed by `LibraryChargeHandler.create_force` and so should be forwarded on. As is coded now there isn't special handling for the `unique` kwarg in `LibraryChargeHandler.create_force` so this isn't doing anything. I think it should be removed but may have overlooked a detail.

Reply:
That sounds correct, thanks for catching this. Lost to history is a period of time in which I confused myself about which kwargs are passed to `find_matches` and `create_force`, and even a brief attempt at refactoring how each handler accepts kwargs.